client/delete_test.go

```diff
@@ -0,0 +1,25 @@
+package client_test
+
+import (
+	"testing"
+
+	"github.com/AirHelp/treasury/client"
+	test "github.com/AirHelp/treasury/test/backend"
+)
+
+func TestDelete(t *testing.T) {
+
+	dummyClientOptions := &client.Options{
+		Backend:      &test.MockBackendClient{},
+		S3BucketName: "fake_s3_bucket",
+	}
+	treasury, err := client.New(dummyClientOptions)
+	if err != nil {
+		t.Error(err)
+	}
+
+	err = treasury.Delete(test.Key1)
+	if err != nil {
+		t.Error(err)
```

Comment:
what are we testing here?

Reply:
I've tried to test whether the function works when there is existing key in treasury.